src/hooks/useKeyboardControls.ts

```diff
@@ -99,7 +102,7 @@ export const useKeyboardControls: ReactHook<UseKeyboardControlsProps, void> = ({
     if (column - 1 < 1 || isSeatTaken(row, column - 1)) return;
 
     takeSeat(row, column - 1);
-    onMove && onMove();
+    onMove && onMove(); // TODO: do we need this here anymore?
```

Comment:
- [ ] [TODO] do we need this here anymore? (probably not)

Reply:
resolved in https://github.com/sparkletown/sparkle/pull/840/commits/0fcc08e4ad8d690b9dcd1d9fdeef07ca79f88caa